src/components/TourList/TourListHeaderSection/TourListHeaderSection.jsx

```diff
@@ -64,13 +65,13 @@ function TourListHeaderSection() {
     let modalComponent;
 
     switch (clickedValue) {
```

Comment:
정말 굳이,,? 지만

```suggestion
    clickedValue === 'date' ? <BsCalender/> : clikedValue === 'price' ? <BsPrice/> : <BsTourType/>
```
이렇게도 바꿀 수 있을까? 이것밖에 생각이 안 나는데 흠,,, switch문이 오히려 더 직관적이라,,